bin/publish.js

```diff
@@ -308,22 +310,41 @@ let cli = readline.createInterface({
   output: process.stdout,
 });
 
-function publishPackage(distTag, otp, tarball) {
+/**
+ * If otp is passed add it as a parameter to the publish command else assume authentication is setup either
+ * as environment variable
+ *
+ * @param {string} distTag - Use this tag on npm for this instance
+ * @param {string} tarball - Path to the tarball
+ * @param {string} otp - Token to make publish requests to npm
+ */
+function publishPackage(distTag, tarball, otp) {
+  let cmd = `npm publish ${tarball} --tag=${distTag} --access=public`;
+
+  if (otp) {
+    cmd += ` --otp=${otp}`;
+  }
+
+  console.log(cmd);
```

Comment:
This log seems like it might get out of sync with the actual command. Do we still want this, and if so, lets use the same line in the exec. It seems like if no otp, cmd will not have it but it will still be in the execWithLog?

Reply:
ahh console log should've been removed. Changed to:

```
function publishPackage(distTag, tarball, otp) {
  let cmd = `npm publish ${tarball} --tag=${distTag} --access=public`;

  if (otp) {
    cmd += ` --otp=${otp}`;
  }

  execWithLog(cmd);
}
```